src/storage/src/write_batch.rs

```diff
@@ -587,6 +587,7 @@ mod tests {
 
         let mut batch = new_test_batch();
         let err = batch.delete(keys).unwrap_err();
+        println!("err: {:?}", err);
```

Comment:
Better to use `error!` macro, and tell what this error is. 
```suggestion
        error!("something went err: {:?}", err);
```

Reply:
This is used to test the print results during debugging. I will delete it:D